### PR TITLE
answers questions

### PR DESCRIPTION
**Pull request by Anddrew914**

### CODE REVIEW

File: study.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ manipulates it by sorting it and then reversing it in place (i.e., modifying the
 reference to the original).
 
 ```ruby
-# your answer here
+[23, 56, 3, 7].reverse
 ```
```
Comment (scottyscripts) [inline study.md:42]: This would reverse it but not in place...

Comment (scottyscripts) [reply study.md:42]: see your next answer!

File: study.md [hunk 2]
```diff
@@ -47,14 +47,14 @@ What does an exclamation point after a method usually signify?  e.g.,
...
+It says to modify the original reference, not create a new one
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+class className
 ```
```
Comment (scottyscripts) [inline study.md:58]: how would we create a new instance of this class? (HINT: use .new)

**Comment by scottyscripts** [comment]
Good work

File: study.md (hunk 1 above)
Comment (Anddrew914) [reply study.md:42]: would it be [23, 56, 3, 7].!reverse ?

Comment (Anddrew914) [reply study.md:42]: AH, just read the next one, got it!

**Pull request closed without merging**